### PR TITLE
Disability deprecate old claim status url

### PR DESCRIPTION
**Pull request by cehsu**
Connects to department-of-veterans-affairs/vets.gov-team#1465

### CODE REVIEW

**Comment by jbalboni** [comment]
@cehsu Searching the repo for `disability-claims/track-claims`, I came up with more links that we should change. Can you do the same and update the ones you find?

**Comment by cehsu** [comment]
Thanks @jbalboni. I've gone through and replaced all instances, as well as implemented a redirect. Your thoughts on these changes would be welcome when convenient.

File: src/js/disability-benefits/routes.jsx
Comment (jbalboni) [inline src/js/disability-benefits/routes.jsx]: Did you test this to see that it works? It's a bit surprising to me that it would work, since the base url is set to `track-claims` and is prepended to urls in the route config here.

**Comment by jbalboni** [comment]
@cehsu It seems like there are some strange changes in the diff. Did you start with master when you created this branch?

File: src/js/disability-benefits/routes.jsx
Comment (cehsu) [inline src/js/disability-benefits/routes.jsx]: Thanks for your question @jbalboni. Navigating to /disability-benefits/track-claims results in the following:
screen shot 2017-05-09 at 10 06 20 am
---------------------------------------------------------------------------------------------
screen shot 2017-05-09 at 10 06 39 am
---------------------------------------------------------------------------------------------
screen shot 2017-05-09 at 10 06 43 am

**Comment by cehsu** [comment]
Thanks @jbalboni, extraneous changes tidied up.

File: src/js/disability-benefits/routes.jsx
Comment (jbalboni) [inline src/js/disability-benefits/routes.jsx]: Nope, that all makes sense, I just wasn't sure that the redirect would actually work given that we use a base url. But it looks like it does, though I think we should change the `from` to:

`from="/disability-benefits/track-claims*"`

So that we catch any of the old routes, since those won't work anymore.

**Review by jbalboni — APPROVED** [review]
This looks good to me. I made one comment about adjusting the redirect path.

Can you wait until after 4pm EST to merge this? That's when our daily deploy cut off is, so doing it after that will give us a day to make sure everything is working in staging before it goes to production.

File: src/js/disability-benefits/routes.jsx
Comment (cehsu) [inline src/js/disability-benefits/routes.jsx]: Thanks for the helpful feedback @jbalboni, implemented in commit f412932.

**Pull request merged**
